src/libpmempool/replica.c

```diff
@@ -1359,8 +1359,18 @@ check_shutdown_state(struct pool_set *set,
 		struct shutdown_state curr_sds;
 		shutdown_state_init(&curr_sds, NULL);
 		for (unsigned p = 0; p < rep->nparts; ++p) {
-			if (shutdown_state_add_part(&curr_sds,
-					PART(rep, p)->path, NULL)) {
+			const char *path = PART(rep, p)->path;
+			/*
+			 * skip missing parts to avoid false positive shutdown
+			 * state failure detection
+			 */
+			const int exists = util_file_exists(path);
+			if (exists < 0)
+				return -1;
+			else (!exists)
```

Comment:
Nice syntax error ;)

But instead of fixing this to use "else if" just use "if".